src/testsuite/vec_int128_dummy.c

```diff
@@ -1172,9 +1178,13 @@ example_print_vint128 (vi128_t value)
   t_high = (vui64_t) vec_srqi (tmpq, shift_ten16);
   tmp = vec_vmuloud (t_high, (vui64_t) mul_ten16);
   t_mid = (vui64_t) vec_subuqm (val128, tmp);
- 
+#ifdef __clang__
```

Comment:
should we add a comment about why this is necessary?

Reply:
// Work-around for GCC PR 96139